### DISCUSSION TITLE
Error: Failed to connect to ReactomeGSA at 'https://gsa.reactome.

### DISCUSSION THREAD

**Discussion by MaDaLong94** [reactome/ReactomeGSA#57] [Q&A] [answered]
> gsva_result <- analyse_sc_clusters(sce, verbose = TRUE)
Calculating average cluster expression...
Converting expression data to string... (This may take a moment)
Conversion complete
Submitting request to Reactome API...
Error: Failed to connect to ReactomeGSA at 'https://gsa.reactome.org/: schannel: failed to receive handshake, SSL/TLS connection failed. Try reverting the ReactomeGSA url to http://gsa.reactome.org using options(reactome_gsa.url = "http://gsa.reactome.org").

How should I fix it?？？？

**Comment by jonoave** [comment]
Just downloaded the package, installation went fine. Also experiencing the same error

````
> gsva_result <- analyse_sc_clusters(dkfz.all, verbose = TRUE)
Calculating average cluster expression...
Converting expression data to string... (This may take a moment)
Conversion complete
Submitting request to Reactome API...
Compressing request data...
No encoding supplied: defaulting to UTF-8.
Error in start_reactome_analysis(request = request, compress = compress,  : 
  Request failed (503 - Service Unavailable): Failed to connect to storage system. Please try again in a few minutes.
````

**Comment by jgriss** [comment, marked as answer]
Hi all,

This issue is now resolved.

It was caused by a significant server failure at our hosting provider. Everything should now be back to normal.

Thanks for your patience!

Kind regards,
Johannes

**Reply by jonoave** [reply]
Thanks for getting this to work again, just before the year-end holidays!
